### DISCUSSION TITLE
.Onion website for revolt.chat!

### DISCUSSION THREAD

**Discussion by FillingTheVoidOnYT** [stoatchat/discussions#438] [Polls]
An onion instance for revolt.chat would be a game changer not only the first discord alternative to have an onion instance it would increase privacy greatly for those users who need it!

**Comment by Inderix** [comment]
Talking about Revolt being used to bypass school blocks is likely not good for the application's image, do note that.

Aside from that, I do like the idea of a Revolt.onion site, I feel that would be something for later on when the platform gets E2EE and is generally more developed.

**Reply by sussycatgirl** [reply]
I don't see how running Revolt as an onion service would help with circumventing a school filter anyway. Once you're connected to the TOR network, it doesn't matter whether the destination is a clearnet or onion service.
